### DISCUSSION TITLE
i have an error

### DISCUSSION THREAD

**Discussion by vattsall** [ConsenSys-archive/truffle#5467] [General Q&A]
i am getting an error while i tried testing my contract 
<img width="287" alt="Capturre" src="https://user-images.githubusercontent.com/69675188/185564662-47e50a50-f6d9-450b-b91e-0cf287bbfed6.PNG">
it works fine on the terminal 
i know it is an error on metamask and ganache but can anyone help me out???please....

**Comment by LoveKosmos** [comment]
Hey @vattsall could you paste the full error response, please? As well as a snippet of the code associated with it?
Do you have any errors in your terminal?

It might be related to MetaMask interaction.

**Reply by vattsall** [reply]
the error code is 32603

**Reply by apdan510** [reply]
@vattsall Is there any more to that error message? does it expand any further? Also could you please include a code snippet of the transaction command?
